Packs/SysAid/Integrations/SysAid/SysAid.py

```diff
@@ -916,6 +923,22 @@ def service_record_attach_file_command(client: Client, args: Dict[str, Any]) ->
     return command_results
 
 
+def service_record_get_file_command(client: Client, args: Dict[str, Any]):
+    sr_id = str(args.get('id'))
+    file_id = str(args.get('file_id'))
+    file_name = str(args.get('file_name'))
+
+    response = client.service_record_get_file_request(sr_id, file_id)
+
+    if response.status_code == 200:
```

Comment:
We don't want to return something if the response status code is not 200?

Reply:
I corrected the integration by adding an error message in the case of else